newIDE/app/src/UI/Search/UseSearchItem.spec.js

```diff
@@ -15,3 +15,55 @@ describe('UseSearchItem', () => {
     ]);
   });
 });
+
+describe('partialQuickSort', () => {
+  test('Sort an empty search result', () => {
+    const elements = [];
+    partialQuickSort(elements, a => a, 1, 0);
+    expect(elements).toStrictEqual([]);
+  });
+  test('Sort an unique search result', () => {
+    const elements = [1];
+    partialQuickSort(elements, a => a, 1, 1);
+    expect(elements).toStrictEqual([1]);
+  });
+  [0.4, 0.6, 1].forEach(pertinence => {
+    test(`Sort search results with same pertinence: ${pertinence}`, () => {
+      const elements = [];
+      elements.length = 8;
+      elements.fill(pertinence);
+      partialQuickSort(elements, a => a, pertinence, pertinence);
+
+      const expectedElements = [];
+      expectedElements.length = 8;
+      expectedElements.fill(pertinence);
+      expect(elements).toStrictEqual(expectedElements);
+    });
+  });
+  test('Sort search results', () => {
+    const elements = [0.2, 0.1, 0.6, 0.4, 1, 0.3, 0.8, 0.5, 0.7, 0.9];
+    partialQuickSort(elements, a => a, 0.1, 1);
+    expect(elements).toStrictEqual(
+      //
```

Comment:
missing comment here and below too

Reply:
It's format comments, but the formatter put them on a new line.

Reply:
`// prettier-ignore` ;)